gallery/src/commonMain/kotlin/com/konyaco/fluent/gallery/App.kt

```diff
@@ -36,10 +37,12 @@ import com.konyaco.fluent.gallery.screen.settings.SettingsScreen
 import com.konyaco.fluent.icons.Icons
 import com.konyaco.fluent.icons.regular.Settings
 import com.konyaco.fluent.surface.Card
+import androidx.compose.ui.window.*
 
 @Composable
-fun App() {
-    Row(Modifier.fillMaxSize()) {
+fun App(titleBarHeight: Dp) {
```

Comment:
Don't forget to update other targets as well 😉

Reply:
> Don't forget to update other targets as well 😉 

Windows support it on Dev branch

Reply:
I meant the app function is used for all targets such as android or web, isn't it?

Reply:
> I meant the app function is used for all targets such as android or web, isn't it?

Yes, i fixed this problem in `sync_master_target` branch.